common/src/autogluon/common/utils/utils.py

```diff
@@ -10,16 +10,34 @@
 logger = logging.getLogger(__name__)
 
 
+def setup_result_path(checkpoint_name):
```

Comment:
Does it need to be in `common`?

Reply:
I'll delete this as this is no longer needed